src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -162,6 +162,21 @@ private static string FindCERToolInInstallLocations()
             }
         }
 
+        internal static void ShowCrashWindow(object sender, CrashPromptArgs args)
+        {
+            var viewModel = sender as DynamoViewModel;
```

Comment:
can you add a null check here?

Reply:
Actually can be null. ShowCrashWindow could be called like ShowCrashWindow(null, new CrashPromptArgs());